packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.ts

```diff
@@ -74,13 +71,20 @@ export interface CoreCommerceFacetOptions {
   >;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   fetchResultsActionCreator: () => AsyncThunkAction<unknown, void, any>;
+  facetResponseSelector: (
```

Comment:
We should test that the use case-specific facet controllers are passing the correct facet selector

We should also test that the core commerce facet controller is using the facet selectors that got passed to it (probably in the #state tests).

Reply:
Done in https://github.com/coveo/ui-kit/pull/3496/commits/e30f4997ba1fc9b669ccea882e4129f81b663d61 and https://github.com/coveo/ui-kit/pull/3496/commits/06d1f67ec53b85539486a41899ea821dd1505c45